HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -1404,9 +1417,16 @@ def from_oga(software_info)
   end
 
   class PowerOutagePeriod < BaseElement
-    ATTRS = [:begin_month, :begin_day, :end_month, :end_day]
+    ATTRS = [:begin_month, :begin_day, :begin_hour, :end_month, :end_day, :end_hour, :natvent]
```

Comment:
Lets use more explicit variable names. `natvent` -> `natvent_availability` here and elsewhere